lib/logstash/inputs/dlq.rb

```diff
@@ -0,0 +1,41 @@
+require 'logstash/namespace'
+require 'logstash/inputs/base'
+require 'logstash-input-dlq.jar'
+
+
+class LogStash::Inputs::Dlq < LogStash::Inputs::Base
+  config_name 'dlq'
```

Comment:
I think we need to be consistent. Elsewhere, we've used the full dead_letter_queue (in core config) and here its DLQ. dead_letter_queue is long, but it's self-explanatory to users who are not familiar with this terminology.